5-appinfra/apps/cymbal-bank/accounts-userservice/envs/development/versions.tf

```
@@ -17,6 +17,13 @@
 terraform {
   required_version = ">= 1.3"
 
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 5, < 7"
+    }
+  }
+
   provider_meta "google" {
     module_name = "blueprints/terraform/terraform-google-enterprise-application:bootstrap/v0.1.0"
```

Would be good to fix these attribution strings at the same time, just a suggestion:

```suggestion
    module_name = "blueprints/terraform/terraform-google-enterprise-application:appinfra/cymbal-bank/development/v0.1.0"
```